prometheus-exporters/esxi-host-exporter/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
please add `{{- if .Values.enabled }}` part in here too, the other components have it.